bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -27,7 +27,10 @@ import edu.gemini.spModel.core.{Angle, MagnitudeBand}
 import scalaz._
 import Scalaz._
 
-object GemsStrategy extends AgsStrategy {
+trait GemsStrategy extends AgsStrategy {
+  // By default use the remote backend but it can be overriden in tests
+  private [impl] implicit val backend:VoTableBackend = RemoteBackend
```

Comment:
It is there so that I don't need to modify the `AgsStrategy` API signature passing the backend. This is used locally on GemsStrategy and it is overridable. I guess I could make it non-implicit by passing it directly to the `search` method